src/api/services/memberInfo/MemberInfoService.ts

```diff
@@ -0,0 +1,168 @@
+import {
+  GithubInfo,
+  StudyData,
+  Member,
+  Cohort,
+  Submission,
+  MemberInfo,
+} from "./types";
+import { IGithubApiClient } from "../github/interfaces";
+import { IMemberInfoService } from "./interfaces";
+import { handleError } from "../../common";
+import { GithubTree } from "../github/types";
```

Comment:
타입이나 인터페이스를 불러올 때는 `import type`을 사용해주세요.

```suggestion
import type {
  GithubInfo,
  StudyData,
  Member,
  Cohort,
  Submission,
  MemberInfo,
} from "./types";
import type { IGithubApiClient } from "../github/interfaces";
import type { IMemberInfoService } from "./interfaces";
import { handleError } from "../../common";
import type { GithubTree } from "../github/types";
```